plugins/org.eclipse.elk.alg.vertiflex/src/org/eclipse/elk/alg/vertiflex/p2relative/OutlineNode.java

```diff
@@ -0,0 +1,94 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Kiel University and others.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * SPDX-License-Identifier: EPL-2.0 
+ *******************************************************************************/
+package org.eclipse.elk.alg.vertiflex.p2relative;
+
+/**
+ * An outline node is the basic building block of outline. It has a position and a reference to the next outline node.
+ *
+ */
+public class OutlineNode {
+    
+    private double relativeX;
```

Comment:
I do not understand by some properties are called relative and absolute at first glnace. Maybe explain it somewhere.